app/scrapers/youtube.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Youtube(Scraper):
+    """Scraper class for Youtube"""
+
+    def __init__(self):
+        self.url = 'https://www.youtube.com/results'
+        self.queryKey = 'search_query'
+
+    def parseResponse(self, soup):
+        """ Parse the response and return list of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
```

Comment:
The returned values are not urls. They are dicts.

I know these changes were not tested under Python 3.